fw/http_sched_ratio.c

```diff
@@ -451,15 +451,12 @@ __tfw_sched_ratio_calc_dynamic(TfwRatio *ratio, TfwRatioData *rtodata,
  *    a large group of servers. Is there a way to detect that and do
  *    a partial recalculation of ratios?
  */
-static inline int
+static int
 __tfw_sched_ratio_get_rtt(size_t si, TfwRatio *ratio, TfwRatioData *rtodata)
 {
 	unsigned int recalc;
 	unsigned int val[T_PSZ] = { 0 };
-	TfwPrcntlStats pstats = {
-		.ith = tfw_pstats_ith,
```

Comment:
Why do you change this?

Reply:
We have tfw_pstats_ith to be hardcoded in several places, so the the only valid value for now can be only tfw_pstats_ith, so I moved it under the hood and removed verification function.